test/e2e/flowaggregator_test.go

```diff
@@ -114,6 +124,11 @@ func TestFlowAggregator(t *testing.T) {
 	defer teardownTest(t, data)
 	defer teardownFlowAggregator(t, data)
 
+	k8sUtils, err = NewKubernetesUtils(data)
+	if err != nil {
+		t.Fatalf("Error when creating KubenetesUtils: %v", err)
```

Comment:
```suggestion
		t.Fatalf("Error when creating Kubernetes utils client: %v", err)
```
there is a typo at Kubernetes